### DISCUSSION TITLE
I cannt connect remote desktop

### DISCUSSION THREAD

**Discussion by Dirtiedd** [ovh/manager#5070] [Q&A] [answered]
i was connected yesterday just fine and same with my other admin, now i cannot connect at all and havent changed any credentials.

**Comment by JeremyDec** [comment]
Hello @Dirtiedd, Can you give more information please ?

What application are you trying to log in ? What is your nichandle ? Did you get some error ?

Thanks

**Comment by Dirtiedd** [comment, marked as answer]
I have fully reconnected I connected through the via internet and noticed
Remote Desktop was turned off so I had to turn it on then connect to Remote
Desktop. Thanks for your guys help though I didn’t know I could connect via
web
